File: app/src/main/java/com/example/frup69513/cv/model/Profil.java
package com.example.frup69513.cv.model;

import com.google.firebase.database.IgnoreExtraProperties;

@IgnoreExtraProperties
public class Profil {

    private String name;
    private String photoUrl;
    private String job;
    private String where;
    private String from;
    private String backgroundUrl;
    private String description;
    private String bornDate;
    private String address;
    private String state;
    private String driversLicense;
    private String mail;

    public Profil(){}

    public String getName() {
        return name;
    }

    public String getPhotoUrl() {
        return photoUrl;
    }

    public String getJob() {
        return job;
    }

    public String getWhere() {
        return where;
    }

    public String getFrom() {
        return from;
    }

    public String getBackgroundUrl() {
        return backgroundUrl;
    }

    public String getDescription() {
        return description;
    }

    public String getAddress() {
        return address;
    }

    public String getBornDate() {
        return bornDate;
    }

    public String getDriversLicense() {
        return driversLicense;
    }

    public String getState() {
        return state;
    }

    public String getMail() {
        return mail;
    }
}

File: app/src/main/java/fr/project/mlignereux/cv/model/Profil.kt
package fr.project.mlignereux.cv.model

import com.google.firebase.database.IgnoreExtraProperties

@IgnoreExtraProperties
data class Profil(val name: String? = null,
                  val photoUrl: String? = null,
                  val job: String? = null,
                  val where: String? = null,
                  val from: String? = null,
                  val backgroundUrl: String? = null,
                  val description: String? = null,
                  val bornDate: String? = null,
                  val address: String? = null,
                  val state: String? = null,
                  val driversLicense: String? = null,
                  val mail: String? = null)
